scripts/install.sh

```diff
@@ -14,6 +14,12 @@ if [[ ${#@} -lt 1 || ${#@} -gt 2 ]]; then
     exit 1
 fi
 
+kubectl get deployment olm-operator -n openshift-operator-lifecycle-manager -o=jsonpath='{.spec}' > /dev/null 2>&1
```

Comment:
Check in all namespaces?

Reply:
This seems more reasonable to me the more I think about it. Ideally, we don't have any downstream-specific knowledge embedded upstream, so checking the openshift-prefixed OLM namespace should be sufficient in the short term to prevent users from installing upstream OLM on managed clusters, but I think we'd like to attempt to start diving out downstream in the next couple of releases.

If we're querying for the olm-operator deployment in all of the namespaces, is there a chance we're casting a wide net here? I doubt there are user-defined deployments that match this name but I'm wondering if we need to also add a label selector.